Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -0,0 +1,176 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Gordon360.Authorization;
+
+namespace Gordon360.Services.RecIM
+{
+    public class TeamService : ITeamService
+    {
+        private readonly CCTContext _context;
+        private readonly IMatchService _matchService;
+
+        public TeamService(CCTContext context, IMatchService matchService)
+        {
+            _context = context;
+            _matchService = matchService;
+        }
+
+        public TeamViewModel GetTeamByID(int teamID)
+        {
+
+            var team = _context.Team
+                .Where(t => t.ID == teamID)
+                .Select(t => new TeamViewModel
+                {
+                    ID = teamID,
+                    Name = t.Name,
+                    Status = _context.TeamStatus
```

Comment:
_context.TeamStatus could be null here. Probably change to
```suggestion
                    Status = _context.TeamStatus?
```

Reply:
I will also have to handle this in my own routes, I will fix this in a future refactoring PR